src/config/ssd.md

```diff
@@ -0,0 +1,153 @@
+# Solid State Drives
+
+Post installation, you will need to enable TRIM for solid state drives. You can
+check which devices allow TRIM by running:
+
+```
+$ lsblk --discard
+```
+
+If DISC-GRAN (discard granularity) and DISC-MAX (discard max bytes) columns are
+non-zero, that means the block device has TRIM support. If your solid state
+drive partition does not show TRIM support, please verify that you chose a file
+system with TRIM support (ext4, btrfs, etc.).
+
+To run TRIM one-shot, you can run
+[`fstrim(8)`](https://man.voidlinux.org/fstrim.8) manually. For example, if your
+/ directory is on a SSD, run the following command to run TRIM one-shot:
+
+> Note: To run fstrim on F2FS file systems, you need to be running kernel 4.19
+> or greater.
+
+```
+# fstrim /
+```
+
+We can automate running TRIM by using cron or by adding the `discard` option to
```

Comment:
Change voice:

"To automate running TRIM, use cron, or add the ..."

Reply:
Oh, actually, i see we go into further details about `discard` below. So just change the voice.